src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx

```diff
@@ -48,9 +48,10 @@ const ProposedSolutionsList: React.FC<ProposedSolutionsListProps> = ({ proposedS
   return (
     <>
       <div className={styles.proposedSolutionsList}>
-        {proposedSolutions.map((proposedSolution, i) => (
-          <ProposedSolutionView key={i} proposedSolution={proposedSolution} />
-        ))}
+        {proposedSolutions.map((proposedSolution, i) => {
```

Comment:
i think the best way to do this would be to make `faded` a property of ProposedSolutionView and then pass it in as !proposedSolution.approved

then just need to implement something like this within the component itself

Reply:
[03afa94](https://github.com/Northeastern-Electric-Racing/FinishLine/pull/611/commits/03afa94e0155c05c8e786234b1faef8413b475f1) (maybe ????)